.gitignore

```diff
@@ -1423,4 +1423,6 @@ events.*
 eval_config.py
 collect_demo_data_config.py
 !ding/**/*.py
-events.*
\ No newline at end of file
+events.*
+
+evogym/*
```

Comment:
why "evogym" should be excluded?

Reply:
yes, will remove it in the final version, I left it because it was convenient to me during development